WordPress/Classes/ViewRelated/Blog/SiteManagement/SettingsTitleSubtitleController.swift

```diff
@@ -0,0 +1,283 @@
+import UIKit
+import Gridicons
+
+
+/// Types the closures than can be provided as completion blocks
+typealias SettingsTitleSubtitleAction = ((SettingsTitleSubtitleController.Data) -> Void)
+
+
+/**
+ Presents a view controller with a textfiled and a textview, that can be used to create / edit a title and subtitle pair.
+*/
+final class SettingsTitleSubtitleController: UITableViewController {
+
+    /// The data to be presented on screen (i.e. title and subtitle).
+    final class Data {
+        var title: String?
+        var subtitle: String?
+
+        init(title: String?, subtitle: String?) {
+            self.title = title
+            self.subtitle = subtitle
+        }
+    }
+
+
+    /// String literals to be presented in an action confirmation alert
+    struct Confirmation {
```

Comment:
`Confirmation` seems a bit generic of a name. It might be nice if it had a bit more context, like `ActionConfirmation` or `DeleteConfirmation` depending if the supported action is intended to be configurable or is supposed to always be a delete action.